internal/controller/bucket/create.go

```diff
@@ -140,6 +140,9 @@ WAIT:
 				// Crossplane skips object forever.
 				delete(bucket.ObjectMeta.Annotations, meta.AnnotationKeyExternalCreatePending)
 
+				// Add labels for the backend
+				bucket.Labels[beName] = "true"
```

Comment:
I think this change has to be applied to Update too.

Reply:
Add in [50a3737](https://github.com/linode/provider-ceph/pull/65/commits/50a37373e36c02f54f002009928887ae4bf8c3c2)